guacamole-ext/src/main/java/org/apache/guacamole/token/StandardTokens.java

```diff
@@ -42,6 +43,16 @@
     public static final String PASSWORD_TOKEN = "GUAC_PASSWORD";
 
     /**
+     * The name of the client token added via addStandardTokens().
+     */
+    public static final String REMHOST_TOKEN = "GUAC_REMHOST";
```

Comment:
I'm not the biggest fan of "GUAC_REMHOST" and "GUAC_REMIP", given the verbosity of other tokens ("GUAC_USERNAME" and "GUAC_PASSWORD"), the ambiguity of "remote" in a multi-layered stack, and the specificity of "IP" compared to the "address" property it's pulled from.

What would you think of "GUAC_CLIENT_HOSTNAME" and "GUAC_CLIENT_ADDRESS"?